lib/src/view/board_editor/board_editor_menu.dart

```diff
@@ -15,6 +15,9 @@ class BoardEditorMenu extends ConsumerWidget {
     final editorController = boardEditorControllerProvider(initialFen);
     final editorState = ref.watch(editorController);
 
+    final enPassantSquares =
+        ref.read(editorController.notifier).calculateEnPassantOptions();
```

Comment:
We usually don't do that in riverpod. It is better to put the en passant options directly in the state instead.